CONTRIBUTING.md

```diff
@@ -0,0 +1,226 @@
+Contributing to BrainVerse
+========================
+This document is adapted from [NICEMAN's CONTRINUTING.md](https://github.com/ReproNim/niceman/blob/master/CONTRIBUTING.md) . Thanks to NICEMAN team!
+
+Files organization
+------------------
+
+- `eapp/` is the folder consisting of source code for the app and where major development is happening.
+- `eapp/` is organized closely as an [Express](https://expressjs.com/en/starter/generator.html) web framework application folder structure:
+    - `config/` - consists of the app configuration file
+    - `modules/` - consists of user interface (UI) of modules that are being actively developed.
+      - Each module source code resides in a separate folder under modules/ so that independent and modularized development can happen.
+      - Current modules that are being actively developed:
+        * `experiment-planner/`
+        * `nda-editor/`
+        * `term-search/`
+        * `audit-trail/`
+      - Deprecated modules:
```

Comment:
shouldn't we remove these?

Reply:
yes, eventually I will be removing these. I have kept these modules for references to some features that were implemented earlier and may need in current UI.